cg/store/models.py

```diff
@@ -895,7 +889,7 @@ def to_dict(self, links: bool = False, flowcells: bool = False) -> dict:
         if links:
             data["links"] = [link_obj.to_dict(family=True, parents=True) for link_obj in self.links]
         if flowcells:
-            data["flowcells"] = [flowcell_obj.to_dict() for flowcell_obj in self.flowcells]
+            data["flowcells"] = [flowcell_obj.to_dict() for flowcell_obj in self.flow_cells]
```

Comment:
for example, here we mix the `flow_cell` terminology with `flowcell`.

Reply:
Since `data["flowcells"]` is exterior-facing (ie the key is probably used in other repos) I don't want to touch that